inc/REST/RemoteDataController.php

```diff
@@ -78,7 +78,8 @@ public static function execute_queries( WP_REST_Request $request ): array|WP_Err
 		);
 	}
 
-	public static function permission_callback(): bool {
-		return true;
+	public static function permission_callback( WP_REST_Request $request ): bool {
+		$post_id = $request->get_param( 'post_id' );
```

Comment:
Should the `post_id` value be santized, before it's sent off to `current_user_can`?

Reply:
Maybe, although it would have to be a generic filter